plugins/scaffolder-backend/src/scaffolder/tasks/TaskWorker.test.ts

```diff
@@ -233,7 +315,7 @@ describe('TaskWorker internals', () => {
             async complete(_result, _metadata) {},
           } as TaskContext;
         },
-      } as TaskBroker,
+      } as unknown as TaskBroker,
```

Comment:
Hmm do we need this?

Reply:
Yeah, as we don't need to define for the test all method definitions. It would be harder to read it then.

It's interesting, that if only claim method is defined, type system is happy

<img width="758" alt="Screenshot 2023-03-08 at 19 54 33" src="https://user-images.githubusercontent.com/2265094/223808330-b0cca44c-2160-4343-bf64-a30c3ca178bf.png">

but if to add also $event, now type checks wants all methods to be defined 🤯 

<img width="713" alt="Screenshot 2023-03-08 at 19 54 42" src="https://user-images.githubusercontent.com/2265094/223808451-93550f39-d6b2-4177-b3ae-b81cdc62e081.png">